src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -47,21 +50,128 @@
+                                :data {:member? (contains? (:my-auth-domains @state) group)
+                                       :requested? false
+                                       :requesting? false}})
+                             (:ws-auth-domains props)))})
+   :render
+   (fn [{:keys [props state this]}]
+     [comps/OKCancelForm
+      {:header "Request Access"
+       :content
+       (react/create-element
+        (let [{:keys [my-auth-domains ws-instructions error]} @state]
+          (cond
+            (not (or (and my-auth-domains ws-instructions) error))
+            [comps/Spinner {:text "Loading authorization domains..."}]
+            error
+            (case (:code error)
+              (:unknown :parse-error)
+              [:div {:style {:color (:exception-state style/colors)}}
+               "Error:" [:div {} (:details error)]]
+              [comps/ErrorViewer {:error (:details error)}])
+            :else
+            [:div {:style {:width 750}}
+             [:div {} "You cannot access this workspace because it contains restricted data.
+                         You need permission from the admin(s) of all of the Authorization Domains
+                         protecting the workspace."]
+             (let [simple-th (fn [text]
+                               [:th {:style {:textAlign "left" :padding "0 0.5rem"
+                                             :borderBottom style/standard-line}} text])
+                   simple-td (fn [text]
+                               [:td {}
+                                [:label {:style {:display "block" :padding "1rem 0.5rem"
+                                                 :width "33%"}} text]])
+                   instructions (into {}
+                                      (map (fn [ad]
+                                             {(keyword (:groupName ad)) (:instructions ad)})
+                                           ws-instructions))]
+               [:form {:style {:margin "1em 0 1em 0"}}
+                [:table {:style {:width "100%" :borderCollapse "collapse"}}
+                 [:thead {}
+                  [:tr {}
+                   (simple-th "Authorization Domain")
+                   (simple-th "Access")
+                   (simple-th "")]]
+                 [:tbody {}
+                  (map-indexed (fn [i auth-domain]
+                                 (let [name (:name auth-domain)
+                                       instruction ((keyword name) instructions)
+                                       member? (:member? (:data auth-domain))
+                                       requested? (:requested? (:data auth-domain))
+                                       requesting? (:requesting? (:data auth-domain))]
+                                   [:tr {}
+                                    (simple-td name)
+                                    (simple-td (if member? "Yes" "No"))
+                                    (if (some? instruction)
```

Comment:
Since Clojure has reasonable semantics around truthy/falsy values, `instruction` is more idiomatic here. Only `nil` and `false` are false.